extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-input.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-recaptcha-input';
+
+describes.realWin('amp-recaptcha-input 0.1 version', {
+  win: { /* window spec */
+    location: '...',
+    historyOff: false,
+  },
+  amp: { /* amp spec */
+    runtimeOn: false,
+    extensions: ['amp-recaptcha-input:0.1'],
+  },
+}, () => {
+
+  describe('amp-recaptcha-input', () => {
+    it('should pass', () => {
```

Comment:
no need for this : )

Reply:
I noticed that if there are no tests, the unit tests will fail saying that Karma could not find any tests. "ERROR: Zero tests detected by Karma. Something went wrong."

See this Travis build's unit tests for a commit that had all of the describes in place: https://travis-ci.org/ampproject/amphtml/jobs/419779541

Reply:
Got it. Thanks for the info